frontend/app/controllers/spree/checkout_controller.rb

```diff
@@ -52,6 +51,26 @@ def update
     end
 
     private
+
+      def update_params
```

Comment:
I'm still not a fan of the fact that we define this twice even though we already have a module in place which we can use to DRY this up.